java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -3689,6 +3690,38 @@ public T functionList(@NonNull String libNamePattern, boolean withCode) {
         return getThis();
     }
 
+    /**
+     * Invokes a previously loaded function.
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://redis.io/docs/latest/commands/fcall/">redis.io</a> for details.
+     * @param function The function name.
+     * @param keys An <code>array</code> of key arguments accessed by the function.
+     * @param arguments An <code>array</code> of <code>function</code> arguments.
+     * @return Command Response - The invoked function's return value.
+     */
+    public T fcall(@NonNull String function, @NonNull String[] keys, @NonNull String[] arguments) {
+        ArgsArray commandArgs =
+                buildArgs(
+                        concatenateArrays(
+                                new String[] {function, Long.toString(keys.length)}, keys, arguments));
+        protobufTransaction.addCommands(buildCommand(FCall, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Invokes a previously loaded function.
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://redis.io/docs/latest/commands/fcall/">redis.io</a> for details.
+     * @param function The function name.
+     * @param arguments An <code>array</code> of <code>function</code> arguments.
+     * @return Command Response - The invoked function's return value.
+     */
+    public T fcall(@NonNull String function, @NonNull String[] arguments) {
```

Comment:
I'm not sure this signature is really that necessary.  
Do we also need to include a signature with zero arguments?  but has keys?  
```
fcall(@NonNull String function, @NonNull String[] keys)
```

although we can't

Reply:
Will be a conflict, need to give them differernt names. TBD
I'm welcome to change that.